opal/mca/threads/pthreads/threads_pthreads_mutex.c

```diff
@@ -0,0 +1,63 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil -*- */
+/*
+ * Copyright (c) 2004-2005 The Trustees of Indiana University and Indiana
+ *                         University Research and Technology
+ *                         Corporation.  All rights reserved.
+ * Copyright (c) 2004-2005 The University of Tennessee and The University
+ *                         of Tennessee Research Foundation.  All rights
+ *                         reserved.
+ * Copyright (c) 2004-2005 High Performance Computing Center Stuttgart,
+ *                         University of Stuttgart.  All rights reserved.
+ * Copyright (c) 2004-2005 The Regents of the University of California.
+ *                         All rights reserved.
+ * Copyright (c) 2007-2016 Los Alamos National Security, LLC.  All rights
+ *                         reserved.
+ * Copyright (c) 2015      Research Organization for Information Science
+ *                         and Technology (RIST). All rights reserved.
+ * $COPYRIGHT$
+ *
+ * Additional copyrights may follow
+ *
+ * $HEADER$
+ */
+
+#include "opal_config.h"
+
+#include <errno.h>
+#include <pthread.h>
+
+#include "opal/mca/threads/mutex.h"
+//#include "opal/mca/threads/pthreads/mutex_unix.h"
```

Comment:
If you don't need this include, please delete the line (vs. commenting it out).

Reply:
This is still an issue.

I've found several of these so far; I'm going to stop marking them.  Please re-examine prior reviews and either fix the outstanding issues or explicitly state "I'm not going to change this, and here's why..." so that we can know you saw the comment and made a choice.  Thanks!